specification/common-types/resource-management/v4/managedidentity.json

```diff
@@ -11,7 +11,8 @@
       "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
       "type": "object",
       "additionalProperties": {
-        "$ref": "#/definitions/UserAssignedIdentity"
+        "$ref": "#/definitions/UserAssignedIdentity",
+        "x-nullable": true
       }
```

Comment:
lets chat about why this is required

Reply:
@rkmanda [MI PATCH](https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html#patch-behavior) As per this Doc, null is an allowed value in user assigned managed identity PATCH payload. And for clients like azure CLI, "x-nullable" is required so that it allows customer to pass null values in the patch payload.
Not sure why this nullable field wasn't required in the other services PATCH payload.
Anyways, since this wasn't a requirement for the other services, we would modify only at our service, instead of common types.

Reply:
I had created a new version for common types and added "x-nullable" field as per the requirement.